src/mirrors/mirrorRegistrar.js

```diff
@@ -44,7 +44,8 @@ DEBUG = !!process.env.VELOCITY_DEBUG;
           DEBUG && console.log('[velocity] Mirror connected to parent. Registering mirror...');
           velocityConnection.call('velocity/mirrors/register', {
             framework: process.env.FRAMEWORK,
-            host: process.env.HOST
+            host: process.env.HOST,
+            port: process.env.MIRROR_PORT
```

Comment:
Velocity environment variables should be prefixed with VELOCITY_. I know that there are other existing ones that don't have the prefix. We will fix this in an upcoming release.